src/main/java/com/hivemq/cli/commandline/CommandErrorMessageHandler.java

```diff
@@ -21,19 +22,21 @@
 import javax.inject.Inject;
 import java.io.PrintWriter;
 
-public class CommandErrorMessageHandler extends CommonErrorMessageHandler implements CommandLine.IParameterExceptionHandler {
+public class CommandErrorMessageHandler extends CommonErrorMessageHandler
+        implements CommandLine.IParameterExceptionHandler {
 
-   @Inject CommandErrorMessageHandler() {};
+    @Inject
+    CommandErrorMessageHandler() {}
 
     @Override
-    public int handleParseException(final @NotNull CommandLine.ParameterException ex, final @NotNull  String[] args) throws Exception {
-
+    public int handleParseException(
```

Comment:
i'd rather would do this:
method(final @NotNull CommandLine.ParameterException ex, 
              final @NotNull String @NotNull [] args) throws ...

Reply:
This is due to auto formatting. Will be hopefully fixed with the unified formatting.